genesis/genesis_fuji.go

```diff
@@ -42,11 +42,15 @@ var (
 					gas.DBWrite:   1,
 					gas.Compute:   1,
 				},
-				MaxCapacity:              1_000_000,
-				MaxPerSecond:             1_000,
-				TargetPerSecond:          500,
-				MinPrice:                 1,
-				ExcessConversionConstant: 5_000,
+				MaxCapacity:     1_000_000, // Max block size ~1MB
+				MaxPerSecond:    250_000,
+				TargetPerSecond: 125_000, // Target block size ~125KB
```

Comment:
For my understanding, what is the relation between block size and gas consumption?